client/src/main/scala/drt/client/components/DropInComponent.scala

```diff
@@ -31,23 +31,20 @@ object DropInComponent extends WithScalaCssImplicits {
       dropInRegistrations = model.dropInRegistrations
     ))
 
-    private def roundToNearestHalfHour(hours: Double): Double = {
-      val wholeHours = hours.toInt
-      val fractionalHours = hours - wholeHours
+    def formatDuration(minutes: Int): String = {
+      val roundedMinutes = (math.round(minutes.toFloat / 15) * 15).toInt
+      val displayMinutes = if (roundedMinutes % 60 < 1) "" else s"${roundedMinutes % 60} minutes"
```

Comment:
This logic isn't right. If the remainder is 1 then it's singular else it's plural. Let's get some tests around this

Reply:
we are not displaying minutes specially singular mins . It would be 15 minutes minimum

Reply:
In that case there's no need for this logic right?
0 minutes
15 minutes
30 minutes
45 minutes

There's no plural case. But let's have tests with some sample inputs and outputs

Reply:
It just we don't want to display 0 minutes